docs/installation.rst

```diff
@@ -4,7 +4,7 @@ Installing
 Which Python?
 -------------
 
-You'll need **Python 3.10 or greater**. 
+You'll need **Python 3.10 up until 3.12**. 
```

Comment:
What about rephrasing it to one of the following: 
- "We support Python 3.10 to 3.12". 
- "The supported python versions are 3.10-3.12"
- "Imod-python has been tested on Python 3.10 to 3.12"
- "Imod-Python has been developed using Python 3.10 to 3.12"
-

Reply:
Changed to most explicit as possible: "You'll need **Python 3.10, 3.11, or 3.12**. "